### PR TITLE
raft_client: calculate batch size more approximately

### PR DESCRIPTION
**Pull request by hicqu**
Signed-off-by: qupeng <qupeng@pingcap.com>

### What problem does this PR solve?

Calculate batch raft message size more approximately. Try best to avoid error like:
```
 ["RPC batch_raft fail"] [err="Some(RpcFailure(RpcStatus { status: ResourceExhausted, details: Some(\"Sent message larger than max (10525503 vs. 10485760)\") }))"] [sink_err=Some(RemoteStopped)] [to_addr=10.240.97.140:20160]
```

### What is changed and how it works?

Enlarge the batch buff from 4k to 64k, and consider size of `RaftMessage` itself.

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- PR to update `pingcap/tidb-ansible`:
- Need to cherry-pick to the release branch

### Check List 

Tests 

- Unit test

Side effects

- Performance regression
    - Consumes more CPU
    - Consumes more MEM
- Breaking backward compatibility

### Release note

### CODE REVIEW

File: src/server/raft_client.rs [hunk 1]
```diff
@@ -243,9 +242,9 @@ impl<T: RaftStoreRouter<RocksEngine>> RaftClient<T> {
 struct RaftMsgCollector(usize);
 impl BatchCollector<Vec<RaftMessage>, RaftMessage> for RaftMsgCollector {
     fn collect(&mut self, v: &mut Vec<RaftMessage>, e: RaftMessage) -> Option<RaftMessage> {
-        let mut msg_size = 0;
+        let mut msg_size = e.start_key.len() + e.end_key.len() + mem::size_of::<RaftMessage>();
```
Comment (BusyJay) [inline src/server/raft_client.rs:245]: Why counting memory size? It has nothing to do with the final result.

Comment (hicqu) [reply src/server/raft_client.rs:245]: Most of fields of `RaftMessage` are inlined, so memory size is close to `compute_size`.

Comment (BusyJay) [reply src/server/raft_client.rs:245]: That's not how compute_size work. For example, zero numbers won't be counted in compute_size.

**Pull request merged**